app/src/main/java/com/sopt/dive/presentation/signin/SignInScreen.kt

```diff
@@ -0,0 +1,182 @@
+import com.sopt.dive.core.designsystem.theme.DiveTheme
+import com.sopt.dive.core.extention.noRippleClickable
+import com.sopt.dive.core.extention.showToast
+import com.sopt.dive.core.util.Validator
+
+@Composable
+fun SignInRoute(
+    onSignUpClick: () -> Unit,
+    onSignInSuccess: () -> Unit
+) {
+    val context = LocalContext.current
+    val userManager = remember { UserManager(context) }
+    val userData = remember { userManager.loadUserData() }
+
+    var id by remember(userData.id) { mutableStateOf(userData.id) }
+    var password by remember(userData.password) { mutableStateOf(userData.password) }
+
+    SignInScreen(
+        id = id,
+        password = password,
+        onIdChange = { id = it },
+        onPasswordChange = { password = it },
+        onLaunchSignUp = onSignUpClick,
+        onSignInSuccess = {
+            val validationError = Validator.validateSignInInputs(id, password)
+            if (validationError != null) {
+                context.showToast(validationError)
+                return@SignInScreen
+            }
+
+            if (id == userData.id && password == userData.password) {
+                context.showToast("로그인에 성공했습니다!")
+                userManager.setAutoLogin(true)
+                onSignInSuccess()
+            } else {
+                context.showToast("아이디 또는 비밀번호가 틀렸습니다.")
+            }
+        }
+    )
+}
+
+@Composable
+private fun SignInScreen(
+    modifier: Modifier = Modifier,
+    id: String,
+    password: String,
+    onIdChange: (String) -> Unit,
+    onPasswordChange: (String) -> Unit,
+    onLaunchSignUp: () -> Unit,
+    onSignInSuccess: () -> Unit
+) {
+    val focusRequesterPassword = remember { FocusRequester() }
+    val focusManager = LocalFocusManager.current
+
+    Column(
+        modifier = modifier
+            .fillMaxSize()
+            .verticalScroll(rememberScrollState())
+            .padding(horizontal = 20.dp)
+            .imePadding(),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.Top
+    ) {
+        Text(
+            text = "Welcome To SOPT",
+            fontSize = 40.sp,
+            fontStyle = FontStyle.Italic,
+            fontWeight = FontWeight.Bold,
+            modifier = Modifier
+                .padding(top = 40.dp)
+        )
+
+        Column(
+            modifier = Modifier.padding(top = 40.dp),
+            verticalArrangement = Arrangement.spacedBy(24.dp)
+        ) {
+            SoptFormField(
+                label = "ID",
+                value = id,
+                onValueChange = onIdChange,
+                placeholder = "아이디를 입력해주세요.",
+                maxLines = 1,
+                keyboardOptions = KeyboardOptions(
+                    keyboardType = KeyboardType.Text,
+                    imeAction = ImeAction.Next
+                ),
+                keyboardActions = KeyboardActions(
+                    onNext = { focusRequesterPassword.requestFocus() }
```

Comment:
```suggestion
               keyboardActions = KeyboardActions(
            onNext = { focusManager.moveFocus(FocusDirection.Down) }
        )
```
아마 next를 눌렀을 때 다음 textfield로 focus가 넘어가길 원해서 이런 코드를 작성한 것 같아요!

Reply:
오 FocusRequester 변수도 필요없어져서 깔끔하고 좋은 것 같아요!